Lombiq.HelpfulLibraries.OrchardCore/Elasticsearch/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.Configuration;
+using Nest;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Search.Elasticsearch.Core.Models;
+
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Is this namespace right?

Reply:
No. I changed the `this` type but forgot to change the namespace too.